docs/sources/enterprise/usage-insights/presence-indicator.md

```diff
@@ -0,0 +1,25 @@
++++
+title = "Presence indicator"
```

Comment:
How about "See who is present" instead of "Presence indicator", which sounds like a robot. We are having to work around this oddity, and let’s see what we can do.

Reply:
I think we should use consistent language ("presence indicator") until we make a product-wide change.